examples/transloadit/main.js

```diff
@@ -1,5 +1,5 @@
-const { inspect } = require('node:util')
-const robodog = require('@uppy/robodog')
+import robodog from 'uppy'
+import 'uppy/dist/uppy.min.css'
```

Comment:
@Murderlon for the migration guide, we could link to the commit where we migrate this example out of Robodog, wdyt?

Reply:
which commit would that be specifically? Current diff is huge, if that one is bite sized it may make sense.

Reply:
https://github.com/transloadit/uppy/pull/3946/commits/29c443e218562cb50f8b5be7e203ada85a39ca0b